tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor.Tests/MockGitHubEventClient.cs

```diff
@@ -455,5 +455,17 @@ public List<GitHubIssueToLock> GetGitHubIssuesToLock()
         {
             return _gitHubIssuesToLock;
         }
+
+        /// <summary>
+        /// Override for the GitHubEventClient funcion which computes this based upon the repository's core rate limit.
+        /// Since this isn't necessary for the tests, just return the ActionRateLimitNonEnterprise/10.
+        /// </summary>
+        /// <returns>int</returns>
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public override async Task<int> ComputeScheduledTaskUpdateLimit()
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
+        {
+            return RateLimitConstants.ActionRateLimitNonEnterprise/10;
```

Comment:
Can you return this value wrapped in a Task so you don't have to set these warnings?

Reply:
I don't see one being better than the other. If this wasn't a mock, I'd probably just do but I don't see a problem with disabling the warning and just returning the value in this case. Also, this isn't the only function that's disabling the warning because the mock doesn't make any async calls but the virtual does. I'm going to leave this as is.